python/tank/pipelineconfig_utils.py

```diff
@@ -183,148 +202,50 @@ def get_core_path_for_config(pipeline_config_path):
 
     In the case of a localized PC, it just returns the given path.
     Otherwise, it resolves the location via the core_xxxx.cfg files.
-    
+
     :param pipeline_config_path: path to a pipeline configuration
 
     :returns: Path to the studio location root or pipeline configuration root or None if not resolved
     """
     try:
-        return _get_core_path_for_config(pipeline_config_path)
-    except:
+        studio_folder = os.path.join(
```

Comment:
wondering about the the `associated_core_descriptor` -- because it's an installed config descriptor, we make the assumption that its core descriptor is always of `path` type - feels like ideally shouldn't make that assumption.

How about for example having a `get_core_path` method which only exists on installed config descriptors? Maybe that could include the `..` logic here too potentially?